src/components/IconButton/IconButton.tsx

```diff
@@ -49,22 +49,22 @@ const sizeStyles = (size: IconButtonProps['size'] = 'mega') => (
   });
 };
 
+const IconButtonComponent = (
+  { children, label, size, ...props }: IconButtonProps,
+  ref?: React.Ref<HTMLButtonElement & HTMLAnchorElement>
+) => {
+  const child = Children.only(children);
+  const icon = cloneElement(child, { role: 'presentation' });
```

Comment:
So we are doing specifically to add the role to children?

Reply:
Yup, that's correct.